cocos2d/core/renderer/webgl/assemblers/motion-streak.js

```diff
@@ -92,14 +92,15 @@ export default class MotionStreakAssembler extends Assembler2D {
         let stroke = comp._stroke / 2;
 
         let node = comp.node;
+        node._updateWorldMatrix();
         let matrix = node._worldMatrix.m;
```

Comment:
`let matrix = node.getWorldMatrix(cache_matrix);`

This will ensure the matrix is always updated, and it avoid using private method